src/public/sass/ITCSS/06_components/pages/product/modal-form/_modal-form.scss

```
@@ -0,0 +1,118 @@
+@use 'sass:color';
+@use '../../../../01_settings/global' as *;
+@use '../../../../01_settings/colors' as *;
+@use '../../../../02_tools/breakpoints' as *;
+
+.modal-form {
+  position: relative;
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  align-items: flex-end;
+  margin: 0 auto;
+
+  &__title {
+    margin: 0 auto;
+    margin-bottom: 1.25rem;
+    color: $brand-color;
+    font-size: 0.9375rem;
+    line-height: 1;
+    white-space: nowrap;
+
+    &--big {
```

```suggestion
    &--large {
```
i think using the word big is not the best option, it would be better if you use large or level 1 or something similar to other examples